packages/utils/src/lib/reports/md/paragraphs.ts

```diff
@@ -1,5 +1,7 @@
+import { NEW_LINE } from './constants';
+
 export function paragraphs(
   ...sections: (string | undefined | boolean)[]
 ): string {
-  return sections.filter(Boolean).join('\n\n');
+  return sections.filter(Boolean).join(NEW_LINE);
```

Comment:
One `\n` isn't enough to separate paragraphs.

```suggestion
  return sections.filter(Boolean).join('\n\n');
```

Reply:
fixed